modules/machineset-yaml-gcp.adoc

```diff
@@ -0,0 +1,85 @@
+/ Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+// * machine_management/creating-machineset-gcp.adoc
+
+[id="machineset-yaml-gcp_{context}"]
+=  Sample YAML for a MachineSet Custom Resource on GCP
+
+This sample YAML defines a MachineSet that runs in Google Cloud Platform (GCP_
+and creates nodes that are labeled with `node-role.kubernetes.io/<role>: ""`
+
+In this sample, `<infrastructureID>` is the infrastructure ID label that is
+based on the cluster ID that you set when you provisioned
+the cluster, and `<role>` is the node label to add.
+
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructureID> <1>
+  name: <infrastructureID>-w-a <2>
+  namespace: openshift-machine-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructureID> <1>
+      machine.openshift.io/cluster-api-machineset: <infrastructureID>-w-a <2>
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructureID> <1>
+        machine.openshift.io/cluster-api-machine-role: worker
+        machine.openshift.io/cluster-api-machine-type: worker
+        machine.openshift.io/cluster-api-machineset: <infrastructureID>-w-a <2>
+    spec:
+      metadata:
+        labels:
+          node-role.kubernetes.io/<role>: "" <3>
+      providerSpec:
+        value:
+          apiVersion: gcpprovider.openshift.io/v1beta1
+          canIPForward: false
+          credentialsSecret:
+            name: gcp-cloud-credentials
+          deletionProtection: false
+          disks:
+          - autoDelete: true
+            boot: true
+            image: <infrastructureID>-rhcos-image
+            labels: null
+            sizeGb: 128
+            type: pd-ssd
+          kind: GCPMachineProviderSpec
+          machineType: n1-standard-4
+          metadata:
+            creationTimestamp: null
+          networkInterfaces:
+          - network: <infrastructureID>-network
+            subnetwork: <infrastructureID>-worker-subnet
+          projectID: openshift-gce-devel
+          region: us-central1
+          serviceAccounts:
+          - email: <infrastructureID>-w@openshift-gce-devel.iam.gserviceaccount.com
```

Comment:
How is this value generated? Do you need to generate a different value for a new machineset up front?